dashboard/src/t5gweb/templates/skeleton.html

```diff
@@ -103,7 +103,8 @@
       Maintained by the Telco5G Field Engineering Team
     </div>
     <div class="d-flex justify-content-between">
-      <div class="muted"><a href="#" data-bs-html="true" data-bs-toggle="tooltip" data-bs-title="1. We update the card data from Jira once an hour
+      <div class="muted"><a href="#" data-bs-html="true" data-bs-toggle="tooltip" data-bs-title="This is not a live view of systems, but rather a copy of relevant data:
```

Comment:
Could we say maybe `but rather a snapshot of the data`:

And minor nit: can you make JIRA all caps?